core/src/com/treode/notify/Notification.scala

```diff
@@ -26,12 +26,13 @@ sealed abstract class Notification [+A] {
 
   def isEmpty: Boolean
   def messages: Seq [Message]
+  override def toString : String
```

Comment:
This is redundant here. toString is already a method of object, which this inherits from.